src/main/java/at/hannibal2/skyhanni/features/inventory/shoppinglist/ShoppingList.kt

```diff
@@ -103,6 +103,44 @@ object ShoppingList {
     var displayItem: ItemStack? = null
 
     // all the functions for interacting with the shopping list come here
+    // parseCommandArguments only works if only the item name is also okay
+    fun parseCommandArguments(args: Array<String>): Triple<String, Double?, String?>? {
```

Comment:
`Triple<String, Double?, String?>`  please  create a new object instead of using a triple